src/config/configuration.ts

```diff
@@ -1,19 +1,40 @@
+import { z } from 'zod';
+
 import { SupportedChainId } from './constants';
 
-export default () => ({
-  about: {
-    name: 'safe-gelato-relay-service',
-  },
```

Comment:
Is this not needed anymore?

Reply:
No, it was used as an example when we first added env var functionality.